filament/backend/include/backend/DriverEnums.h

```diff
@@ -590,7 +590,7 @@ struct SamplerParams { // NOLINT
 
             uint8_t padding2                : 8;    //!< reserved. must be 0.
         };
-        uint32_t u{};
+        uint32_t u = 0;
```

Comment:
out of curiosity, this didn't work with `{}` but worked with `=0`?

Reply:
Well with these last changes in vec I now have new errors, so I cannot try these.

Before your last vec changes, I tried removing the `#if 0` and I got errors. Same for removing the explicit in Quaternion, but not sure about TVec4.

I have not tried again with `u{}`. Is there a reason to prefer this form rather than `u = 0;`?

Reply:
no, just curious

Reply:
Can you revert this? For some reason, I think this breaks the non-MSVC builds.
The `u{}` syntax does a zero-initialization, where as the `={0}` syntax does a value-initialization, I believe; they're subtly different.